code/modules/cargo/packs/livestock.dm

```diff
@@ -190,6 +190,16 @@
 	contains = list(/mob/living/simple_animal/hostile/retaliate/snake = 3)
 	crate_name = "snake crate"
 
+/datum/supply_pack/critter/amphibians
+	name = "Amphibian Friends Crate"
+	desc = "Two disgustingly cute slimey friends. Cytologists love them! \
+		Contains one frog and one axolotl. Warning: Frog may have hallucinogenic properties."
+	cost = CARGO_CRATE_VALUE * 4
+	contains = list(/mob/living/basic/axolotl,
+					/mob/living/basic/frog,
+					)
```

Comment:
```suggestion
	contains = list(
	/mob/living/basic/axolotl,
	/mob/living/basic/frog,
	)
```

your head was in the right place but your indentation was quite odd

Reply:
No errors, nothing in my git. Ran on test server and worked fine. Absolutely no clue otherwise. I copy pasted from another crate that had exact same indentation.

Reply:
i got smartphowned by github webeditor